content/blog/news/CloudPlugin-GSOC'20.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Google Summer of Code '20 - Cloud Plugin"
+linkTitle: "GSoC'20 : Cloud Plugin"
+date: 2020-08-27
+weight: 1
+description: >
+  A starter to the CueCommander cloud plugin; work done for Google Summer of Code '20
+---
+
+A blog post + starter material for the Cloud Plugin - work done during Google Summer of Code '20 on creating a CueCommander
+ plugin that can be used to manage cloud pools from OpenCue
+ 
+## Summer of Code Proposal: 
+
+The main proposal consisted of the plan of action for the 3 months of development, which included research about 
+finding a common language among cloud providers when it came to managing a pool of instances and the ability to create, delete 
+and resize these cloud groups through their API from the OpenCue interface
+
+I took Google Cloud Platform as the primary cloud provider and tried to find parallels in other providers and 
+started designing the main API around it. 
+
+A [Design Document](https://docs.google.com/document/d/1cCPrZsM8fRbcluTppcuPFQ6_tfQo5rZE9W2m6Hms8pw/edit?usp=sharing) 
+about the initial design was then shared with the community for feedback and review which included the specs of the API
+ (which changed over the course of 3 months) and the API matrix which shows what the parallels in each provider are
+
+ 
+## Cloud plugin source code
+
+The code base for this plugin is currently on the [cloudplugin](https://github.com/AcademySoftwareFoundation/OpenCue/tree/cloud-plugin) 
+branch of the OpenCue repo
+
+## Overview of the cloud api design and the plugin interface design
```

Comment:
Try to keep headers short -- I'd just call this one "Design overview". If you want to say more about the contents of the section, add a sentence or two of normal text below the header.